drivers/modem/wncm14a2a.c

```diff
@@ -464,7 +464,7 @@ static int pkt_setup_ip_data(struct net_pkt *pkt,
 
 #if defined(CONFIG_NET_IPV6)
 	if (net_pkt_family(pkt) == AF_INET6) {
-		if (net_ipv6_create_new(
+		if (net_ipv6_create(
 			    pkt,
 			    &((struct sockaddr_in6 *)&sock->src)->sin6_addr,
```

Comment:
Oh, I meant to also point out the net_ipv6_create params (src and dst) are flipped here.  Vs the net_ipv4_create() call right below.  Was in your previous patch series which has already been merged.  (Found it while testing)

Reply:
Go ahead, get a dedicated PR for it as it is a bug fix so it can be applied asap (this current PR has tsc label so it will wait 3 days still)

Reply:
Will do.